### ISSUE TITLE
Question of the learnable matrix of 3D Fourier positional embeddings

### ISSUE THREAD

**Issue by TianhangXiang** [Open3DA/LL3DA#13]
Thank you for the great work!
 In the LL3DA paper, the matrix B of the click position is notated as learnable. But in the code, the positional embeddings are within "with torch.no_grad", it seems that there is no gradient for the matrix, do I miss something, or does keeping the matrix frozen lead to a better result?

**Comment by ch3cook-fdu** [comment]
You are right, the gaussian B matrix are randomly initialized ($\mathcal{N}(0, 1)$), and kept frozen the whole time: https://github.com/Open3DA/LL3DA/blob/main/models/ll3da/position_embedding.py#L36. You are welcome to try training these parameters and see whether they could lead to better performance.

**Comment by TianhangXiang** [comment]
> You are right, the gaussian B matrix are randomly initialized ($\mathcal{N}(0, 1)$), and kept frozen the whole time: https://github.com/Open3DA/LL3DA/blob/main/models/ll3da/position_embedding.py#L36. You are welcome to try training these parameters and see whether they could lead to better performance.

Thank you for your response!
